mage_ai/api/policies/FilePolicy.py

```diff
@@ -63,6 +63,7 @@ class FilePolicy(BasePolicy):
         'exclude_dir_pattern',
         'exclude_pattern',
         'flatten',
+        'include_pipeline_count',
```

Comment:
do you need to add the "pipeline_count" field to the presenter/policy?

Reply:
It seemed to work okay without any 403 errors, but I added it for completeness.